_scripts/icon_list_refresh.sh

```diff
@@ -1,9 +1,18 @@
 #!/bin/bash
 
+if [ -z $1 ]
```

Comment:
Happy to review the script, but is this related to adding tinydrive starters to the nav or should it be in a separate PR?

Reply:
Hi Neil,

This script was added by PR https://github.com/tinymce/tinymce-docs/pull/1618, I accidentally created the PR on the develop branch, not master. This script is for generating the icon list in the docs, it's destructive because if an icon is removed, we need to remove it from the docs (and might also allow us to see when one is accidentally removed from or changed in the TinyMCE repo).